keyboards/skyloong/gk61/pro_ds/pro_ds.c

```diff
@@ -0,0 +1,115 @@
+// Copyright 2023 NaturalZh (@NaturalZh)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+
+#if defined(RGB_MATRIX_ENABLE)
+const aw20216s_led_t PROGMEM g_aw20216s_leds[AW20216S_LED_COUNT] = {
+/* Each AW20216 channel is controlled by a register at some offset between 0x00
+ * and 0xD7 inclusive.
+ * See drivers/awinic/aw20216.h for the mapping between register offsets and
+ * driver pin locations.
+ *   driver
+ *   |  R location
+ *   |  |          G location
+ *   |  |          |          B location
+ *   |  |          |          | */
+    {0, CS1_SW1,   CS2_SW1,   CS3_SW1},
```

Comment:
SWx and CSx need to be swapped around eg. `CS2_SW1` -> `SW1_CS2`.

Reply:
OK, I modified it. Please check again @fauxpark